devnet/docker-compose-substrate.yml

```diff
@@ -8,8 +8,8 @@ services:
         DD_API_KEY: ${DD_API_KEY}
     ports:
       - 7001
-      - 8545
-      - 8546
+      - 8545:8545
+      - 8546:8546
```

Comment:
Containers can reach other without port publishing if they're in the same Docker network, publishing ports is only useful if you want to access it i.e. from your host / terminal.

If these ports are there only for container comm, I'd suggest to change that `ports:` to `expose:` so it's just for documentation and doesn't publish ports to the host.

Reply:
oh actually that is totally what i did it for. I was running a local script against the container. 

My thoughts: I think I leave it since I think for my txn testing it will be largely local, then can unexposed when my tests are migrated to a container.

Open to thoughts if anyone thinks otherwise